src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/UnixExportProvider.cs

```diff
@@ -197,25 +197,26 @@ private byte[] ExportPfx(SafePasswordHandle password)
                 attrBytes[1] = sizeof(int);
                 MemoryMarshal.Write(attrBytes.AsSpan(2), ref keyIdx);
 
+                AttributeAsn attribute = new AttributeAsn
+                {
+                    AttrType = new Oid(Oids.LocalKeyId, null),
+                    AttrValues = new ReadOnlyMemory<byte>[]
+                    {
+                        attrBytes,
+                    }
+                };
                 keyBags[keyIdx] = new SafeBagAsn
                 {
                     BagId = Oids.Pkcs12ShroudedKeyBag,
                     BagValue = ExportPkcs8(certPal, passwordSpan),
                     BagAttributes = new[]
                     {
-                        new AttributeAsn
-                        {
-                            AttrType = new Oid(Oids.LocalKeyId, null),
-                            AttrValues = new ReadOnlyMemory<byte>[]
-                            {
-                                attrBytes,
-                            }
-                        }
+                        attribute
                     }
```

Comment:
Either

```C#
BagAttributes = new[] { attribute },
```

or


```C#
BagAttributes = new[]
{
    attribute,
},
```

, please.  (That's "Add the trailing comma, or collapse it to one line")